SYCL/DeprecatedFeatures/get-options.cpp

```diff
@@ -3,8 +3,10 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
-// XFAIL: cuda || hip
-// XFAIL: ze_debug-1,ze_debug4
```

Comment:
@bso-intel: I saw that you recently added the XFAIL for "ze_debug".  I assume it makes sense to remove this now because this test is now unsupported on Level Zero.  Agree?

Reply:
Yes, I am okay to remove those XFAIL lines as long as Sergey agrees with // UNSUPPORTED: level_zero